### PR TITLE
Follow the mouse

### PR DESCRIPTION
**Pull request by thongncvn**
In this pull request:
- Test to see how fast could Rx subscribing process be.

How to make the red circle move as fast as the mouse pointer do?

### CODE REVIEW

File: apps/follow_the_mouse/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+(function () {
+    var mousePositionStream = Rx.Observable.fromEvent(document, 'mousemove').map(function(event) {
+        return [event.clientX, event.clientY];
+    });
+
```
Comment (nongdenchet) [inline apps/follow_the_mouse/main.js]: Its better to write code like this:

var mousePositionStream = Rx.Observable
    .fromEvent(document, 'mousemove')
    .map(function(event) {
        return [event.clientX, event.clientY];
    });

File: apps/follow_the_mouse/main.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    var mouse = document.querySelector('#mouse');
+    var mousePositionObserver = mousePositionStream.subscribe(function(position) {
+        mouse.style.transform = 'translate(' + position[0] + 'px, ' + position[1] + 'px)';
+    })
+}());
```
Comment (nongdenchet) [inline apps/follow_the_mouse/main.js]: ``
var mousePositionObserver = mousePositionStream
    .subscribe(function(position)  {
        mouse.style.transform = 'translate(' + position[0] + 'px, ' + position[1] + 'px)';
    })

**Comment by nongdenchet** [comment]
Please resolve conflict (I have merge #1 )

**Comment by thongncvn** [comment]
Thank you very much @nongdenchet, I'll edit right now :heart:

**Pull request merged**